nephio/optional/nephio-stock-repos/README.md

```diff
@@ -0,0 +1,21 @@
+# nephio-stock-repos
```

Comment:
@arora-sagar these stock repos are installed with [Ansible instructions in the sandbox](https://github.com/nephio-project/test-infra/blob/main/e2e/provision/playbooks/roles/install/tasks/main.yml#L78-L96), I think that we opt to use kpt packages as long as we can have some level of flexibility.

Reply:
@electrocucaracha I think it is better to use from here rather than the ansible it is twice doing the same work I suppose. I know we do it via there but directory is always fixed `/`. But after this sort of catalog repo structure we have to make it variable too.

Reply:
I see, the only feature that we'll lose is the capability to chose different branches, maybe something that we will have to figure out using kpt CLI instructions.